config/crd/bases/psmdb.percona.com_perconaservermongodbbackups.yaml

```diff
@@ -147,6 +147,39 @@ spec:
               latestRestorableTime:
                 format: date-time
                 type: string
+              minio:
+                properties:
+                  bucket:
+                    type: string
+                  credentialsSecret:
+                    type: string
+                  debugTrace:
+                    type: boolean
+                  endpointUrl:
+                    type: string
+                  forcePathStyle:
+                    type: boolean
+                  insecureSkipTLSVerify:
+                    type: boolean
+                  partSize:
+                    format: int64
+                    type: integer
+                  prefix:
+                    type: string
+                  region:
```

Comment:
Please recheck this task https://perconadev.atlassian.net/browse/K8SPSMDB-1524?focusedCommentId=432562. I am not sure about this new type (please try to reproduce it), but in the case of the S3 type, the users will have big problems without a region. Maybe we need to have required for this option.